docs/source/format/Columnar.rst

```diff
@@ -688,11 +687,10 @@ will have the following layout: ::
             |------------|-----------------------|
             | joemark    | unspecified (padding) |
 
-Similar to structs, a particular child array may have a non-null slot
-even if the validity bitmap of the parent union array indicates the
-slot is null.  Additionally, a child array may have a non-null slot
-even if the types array indicates that a slot contains a different
-type at the index.
+Note that a child array may have a non-null slot even if the types array
+indicates that a slot contains a different type at the index, so the
```

Comment:
I have a little trouble parsing this language.  Could it be stated more as a positive?

> Only the slot in the array corresponding to the type index is considered.  All "unselected" values are ignored and could be any semantically correct array value.

Reply:
OK